ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/infrastructure/deduplication/CommandDeduplicationBase.scala

```diff
@@ -330,6 +332,33 @@ private[testtool] abstract class CommandDeduplicationBase(
     }
   })
 
+  test(
+    s"${testNamingPrefix}DeduplicateParallelSubmissions",
+    "Commands submitted at the same, in parallel, should be deduplicated",
+    allocate(SingleParty),
+  )(implicit ec => { case Participants(Participant(ledger, party)) =>
+    lazy val request = ledger
+      .submitRequest(party, DummyWithAnnotation(party, "Duplicate").create.command)
```

Comment:
Out of curiosity: Where do we actually set the command ID in this test (so that all those commands have the same change ID)?

Reply:
the `submitRequest` call actually creates a request, which internally uses a generated command id. We resubmit the same one so they all have the same command id.

Reply:
:+1: